### PR TITLE
add plugin configuration for "react-native-gesture-handler@1.4.1"

### PR DESCRIPTION
**Pull request by deepueg**
This PR adds support for react-native-gesture-handler. 

https://kmagiera.github.io/react-native-gesture-handler/docs/getting-started.html
https://github.com/kmagiera/react-native-gesture-handler.

Android requires additional [code change](https://kmagiera.github.io/react-native-gesture-handler/docs/getting-started.html#android) on the delegate level once the package is added. This can be added inside a new delegate inside [ern-navigation-impl](https://github.com/electrode-io/ernnavigation-api-impl-native)

#115 
The proposed solution in the above [PR](#115 ) with replaceInFile can be dangerous as it changes the behavior of the base code and there is no easy way to make sure the file does not have compilation issues after the change. Also, if a plugin makes a change to the base file it can have an undesirable impact on other apps/components that do not intend to use this plugin.  Please see the [Readme](https://github.com/electrode-io/electrode-native-manifest/pull/136/files#diff-a31d85a23879610a2c6a5e10509f8874) section to see how the root view creation can be overridden.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -970,6 +970,7 @@
       "lottie-react-native@2.6.1",
       "react-native-android-wifi@0.0.41",
       "react-native-fs@2.13.3",
+      "react-native-gesture-handler@1.4.1",
```
Comment (friederbluemle) [inline manifest.json]: Trailing whitespace here. Please remove :)

**Comment by friederbluemle** [comment]
Merged while reviewing :P

Also useful extension: https://chrome.google.com/webstore/detail/render-whitespace-on-gith/ifdbipohclgnokjgpejhnbjdlgjkkhom?hl=en

**Comment by friederbluemle** [comment]
Also - Did this PR supersede and close #115?

**Comment by deepueg** [comment]
> Merged while reviewing :P
> 
> Also useful extension: https://chrome.google.com/webstore/detail/render-whitespace-on-gith/ifdbipohclgnokjgpejhnbjdlgjkkhom?hl=en

Sorry, lets self add to the reviewer next time. Helps to know when reviewing.  :-)

**Comment by deepueg** [comment]
> Also - Did this PR supersede and close #115?

Yes, it does.

**Comment by friederbluemle** [comment]
> > Merged while reviewing :P
> > Also useful extension: [chrome.google.com/webstore/detail/render-whitespace-on-gith/ifdbipohclgnokjgpejhnbjdlgjkkhom?hl=en](https://chrome.google.com/webstore/detail/render-whitespace-on-gith/ifdbipohclgnokjgpejhnbjdlgjkkhom?hl=en)
> 
> Sorry, lets self add to the reviewer next time. Helps to know when reviewing. :-)

Yeah no worries.. I wish GitHub would somehow make it visible in the PR "Review in progress". ;)

**Pull request merged**